change/@fluentui-react-native-tester-a0ed0eb4-ec38-4450-ae20-c0a1773ba265.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Adding tests to ensure JS + Native stay in sync",
```

Comment:
nit `Add tests to codify testing requirements` I'd avoid references to `native` that could be misleading, an opaque testing requirement is sufficient and we're running the "open all component test pages" test here in the repo too -- native isn't a necessary consideration it's just a policy decision

Reply:
Got it, changing in newest commit